docs/events/gsoc/2024/gradle-build-server-android.md

```diff
@@ -1,3 +1,66 @@
-# GSoC 2024. Gradle Build Server – support for Android projects
```

Comment:
I would keep it, because it will be used for opengraphs&co. But no strong opinion